modules/@apostrophecms/home-page/index.js

```diff
@@ -5,6 +5,16 @@ module.exports = {
   },
   fields: {
     add: {
+      singleton: {
```

Comment:
not used.

Reply:
Yeah singleton is no longer a thing.

Reply:
Oh you gave it the *name* singleton. I guess you could do that but it sounds like this field was left in by accident.